script/npm_link_vendor.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+rm -rf ./node_modules/@theforeman
+mkdir -p ./node_modules/@theforeman
+
+ln -s ../../../foreman-js/packages/vendor-core ./node_modules/@theforeman/vendor-core
```

Comment:
can this be configurable somehow? or at least print reasonable error messages regarding where it expects the directories to be at?

Reply:
Yes!

I will allow configuring via env variables and will add some documentations around it.